release.json

```diff
@@ -1,7 +1,12 @@
 {
   "nio/DynamicFields": {
     "language": "Python",
-    "version": "0.1.1",
-    "url": "git://github.com/nio-blocks/dynamic_fields.git"
+    "url": "git://github.com//nio-blocks/dynamic_fields.git",
```

Comment:
This also has an extra `/` before `nio-block`. I was noticing this sometimes with `nio buildrelease`. It's possible I'm not using the latest version of the cli...